src/languages/r.coffee

```diff
@@ -0,0 +1,35 @@
+# Get Atom defaults
+scope = ['text.html']
+tabLength = atom?.config.get('editor.tabLength', scope: scope) ? 4
+softTabs = atom?.config.get('editor.softTabs', scope: scope) ? true
+defaultIndentSize = (if softTabs then tabLength else 1)
+defaultIndentChar = (if softTabs then " " else "\t")
+defaultIndentWithTabs = not softTabs
+
+module.exports = {
+
+  name: "R"
+  namespace: "r"
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "R"
+]
+
+  ###
+  Supported extensions
+  ###
+  extensions: [
+    "r"
+    "R"
+  ]
+
+  options:
+    indent_size:
```

Comment:
I apologize, I should have been more specific. I meant the user documentation found at https://github.com/Glavin001/atom-beautify/blob/master/docs/options.md

Run [`npm run docs`](https://github.com/Glavin001/atom-beautify/blob/master/package.json#L273) which in turn runs `coffee docs/` and builds the markdown files in the `docs/` directory as well as `README.md` with all of the updated usage information.
However, this does not affect the Atom-Beautify package options, as shown in your screenshot.

I think the problem you are showing above is because of the pre-computed options. You can run [`npm run postinstall`](https://github.com/Glavin001/atom-beautify/blob/master/package.json#L271) which runs `node script/build-options.js`. This will rebuild the file `src/options.json` which is used for package settings, as shown in your screenshot. Afterwards restarting Atom should resolve the problem shown in your screenshot above.